From 782fa12eefe4f5f88d2ed285dc6e7939a82af11a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Tue, 29 Oct 2024 16:02:21 +0200
Subject: [PATCH] commit: use %llu and unsigned long long in printf instead of
 off_t

works better cross architectures

(cherry picked from commit b05ac8826fa0c9b237e99501095b8673050ea81e)
---
 src/commit.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/commit.c b/src/commit.c
index e82b3ddeae8..8d7e669a352 100644
--- a/src/commit.c
+++ b/src/commit.c
@@ -391,16 +391,16 @@ all_done:
 		}
 
 		if (apk_verbosity > 1) {
-			apk_message("%s %d packages, %d dirs, %d files, %zu MiB",
+			apk_message("%s %d packages, %d dirs, %d files, %llu MiB",
 				    buf,
 				    installed_packages,
 				    db->installed.stats.dirs,
 				    db->installed.stats.files,
-				    installed_bytes / (1024 * 1024));
+				    (unsigned long long)installed_bytes / (1024 * 1024));
 		} else {
-			apk_message("%s %zu MiB in %d packages",
+			apk_message("%s %llu MiB in %d packages",
 				    buf,
-				    installed_bytes / (1024 * 1024),
+				    (unsigned long long)installed_bytes / (1024 * 1024),
 				    installed_packages);
 		}
 	}
-- 
2.46.2

